blocks/embed/embed.js

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
This block does not have a JSON definition in the da-block-collection. Will need to add it ourselves.